### PR TITLE
WIP: Refactor controller runtime

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

File: pkg/operator/reconciler.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+)
+
+const ClusterOperatorName = "node-tuning"
+
+// Reconciler reconciles a Tuned object and creates the tuned daemon set
```
Comment (yanirq) [inline pkg/operator/reconciler.go:26]: This is ment for the CO and not tuned object I assume

File: pkg/operator/reconciler.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+	// TODO: it unclear what actions should we do once the ClusterOperator was deleted
+	if instance.DeletionTimestamp != nil {
+		return reconcile.Result{}, nil
+	}
+
+	return reconcile.Result{}, nil
+}
+
+func (r *Reconciler) syncDaemonSet() {
```
Comment (yanirq) [inline pkg/operator/reconciler.go:127]: this should be removed from the current controller and handled in the CVO controller ?

**Comment by yanirq** [comment]
@cynepco3hahue  you can close this PR in favor of https://github.com/openshift/cluster-node-tuning-operator/pull/302

**Comment by cynepco3hahue** [comment]
Close in favor of #302

**Pull request closed without merging**